applications/ContactStructuralMechanicsApplication/tests/ALM_frictional_contact_test_2D/hyper_simple_patch_test_parameters.json

```diff
@@ -83,8 +83,8 @@
         "process_name"          : "ALMContactProcess",
         "Parameters"            : {
             "model_part_name"             : "Structure",
-            "assume_master_slave"         : "Parts_Parts_Auto2",
-            "contact_model_part"          : ["Contact_Part"],
+            "assume_master_slave"         : { "0" : ["Parts_Parts_Auto2"]},
```

Comment:
have you tested with 0 to 9 parts?

Reply:
with 4 parts, but it is supposed to work anyway